lib/api.js

```diff
@@ -71,8 +62,8 @@ api.declare({
 });
 
 api.declare({
-  method:      'patch',
-  route:       '/name/:name(*)',
+  method:      'post',
+  route:       '/secrets/:name(*)',
```

Comment:
/secrets/ is more rest like (I think, I'm no purist)...

I'm not sure about patch, which is why I'm tempted to go for `post` instead..